app/views/ping/index.text.erb

```diff
@@ -0,0 +1,6 @@
+<% content_for(:title){ "Ping" } -%>
+Madeline
+App is running.
+Delayed Job is <%= @dj ? "" : "NOT " %>running.
+<%= "There are no stuck jobs." if !@stuck %>
+<%= "Everything is OK." if @ok %>
```

Comment:
Perhaps this would be a bit better for the last two lines:

```
There <%= @stuck ? "ARE" : "are no" %> stuck jobs.
Everything is <%= @ok ? "OK" : "NOT OK" %>.
```